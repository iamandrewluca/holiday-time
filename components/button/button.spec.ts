import { describe, it, expect } from '@jest/globals'
import { render } from '@testing-library/vue'
import Button from './button.component.vue'

describe('Button', () => {
  it('should render and match snapshot', () => {
    const { baseElement } = render(Button)
    expect(baseElement).toMatchSnapshot()
  })
})
