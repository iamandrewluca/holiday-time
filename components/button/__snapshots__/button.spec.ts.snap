// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render and match snapshot 1`] = `
<body>
  <div>
    <button
      class=""
    />
  </div>
</body>
`;
